<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassicPlayer</class>
 <widget class="QMainWindow" name="ClassicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>412</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Moon Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="playerLayout">
      <property name="spacing">
       <number>6</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="toolbar" native="true">
      <layout class="QHBoxLayout" name="toolbarLayout" stretch="0,0,0,1,0,0,0">
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="playButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="icon">
          <iconset theme="media-playback-start">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="icon">
          <iconset theme="media-playback-pause">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="icon">
          <iconset theme="media-playback-stop">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="progressBar">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="volumeButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="icon">
          <iconset theme="audio-volume-high">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="volumeSlider">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="netButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="icon">
          <iconset theme="applications-internet">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open(&amp;O)</string>
    </property>
    <addaction name="actionAdd_file_s"/>
    <addaction name="actionAdd_url"/>
    <addaction name="actionAdd_playlist"/>
   </widget>
   <widget class="QMenu" name="menuTools_T">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="actionOnline_videos"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAbout_A">
    <property name="title">
     <string>About(&amp;A)</string>
    </property>
    <addaction name="actionPlugins"/>
    <addaction name="actionBrowser_extension"/>
    <addaction name="actionContribute"/>
    <addaction name="actionHomepage"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuTools_T"/>
   <addaction name="menuAbout_A"/>
  </widget>
  <action name="actionAdd_file_s">
   <property name="text">
    <string>Add file(s)</string>
   </property>
  </action>
  <action name="actionAdd_url">
   <property name="text">
    <string>Add url</string>
   </property>
  </action>
  <action name="actionAdd_playlist">
   <property name="text">
    <string>Add playlist</string>
   </property>
  </action>
  <action name="actionOnline_videos">
   <property name="text">
    <string>Online videos</string>
   </property>
  </action>
  <action name="actionTranscoder">
   <property name="text">
    <string>Transcoder</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionBrowser_extension">
   <property name="text">
    <string>Browser extension</string>
   </property>
  </action>
  <action name="actionHomepage">
   <property name="text">
    <string>Homepage</string>
   </property>
  </action>
  <action name="actionContribute">
   <property name="text">
    <string>Contribute</string>
   </property>
  </action>
  <action name="actionPlugins">
   <property name="text">
    <string>Plugins</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
